resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Service_MARIONETTECOLLECTIVE">
```

Comment:
@melissa Yes - thats where :id is used.
I was going to change :id to Service_(gsubbed stuff), but after looking at Sean's code, he can also make use of the :id if we leave it without a Service_ prefix.